pkg/controller/resourceclaim/controller.go

```diff
@@ -841,7 +762,7 @@ func (ec *Controller) syncClaim(ctx context.Context, namespace, name string) err
 		claim := claim.DeepCopy()
 		claim.Status.ReservedFor = valid
 
-		// When a ResourceClaim uses delayed allocation, then it makes sense to
+		// DRA is now always using delayed allocation. Then it makes sense to
 		// deallocate the claim as soon as the last consumer stops using
 		// it. This ensures that the claim can be allocated again as needed by
 		// some future consumer instead of trying to schedule that consumer
```

Comment:
```suggestion
		// DRA always performs delayed allocations. Relatedly, it also
		// deallocates a claim as soon as the last consumer stops using
		// it. This ensures that the claim can be allocated again as needed by
		// some future consumer instead of trying to schedule that consumer
```

Reply:
Applied.